src/battlearena/ChallengePlayerPopup.js

```diff
@@ -0,0 +1,71 @@
+//@flow strict
+
+import * as React from 'react';
+import Popup from 'reactjs-popup';
+import { Link } from 'react-router-dom';
+
+type Props = {|
+	playerChallenged: string,

```

Comment:
Can we get a player class that we pass around? We are going to need more than the players name in order to do these API calls (namely, we will need some sort if playerID), so we want that to get passed around.